### DISCUSSION TITLE
Browser Mod has to be installed?

### DISCUSSION THREAD

**Discussion by Piero87** [matt8707/hass-config#151] [Q&A] [answered]
I see in the documentation that you write avout browser-mod but I don't see it here:
 
<img width="588" alt="Screenshot 2023-01-14 alle 22 22 42" src="https://user-images.githubusercontent.com/768952/212497538-4b3defb6-9295-4c93-9aa6-eafb26f148c6.png">

so my question is has to be installed or not? 

Thank you for this great repo, it's very usefull

**Comment by matt8707** [comment, marked as answer]
if you want popups, yeah

<img width="553" alt="s" src="https://user-images.githubusercontent.com/36163594/212498201-84bdd27e-89dc-4946-8d40-17adf5cbcd00.png">

**Reply by Piero87** [reply]
Thank you
